aloscene/renderer/renderer.py

```diff
@@ -220,8 +251,10 @@ def render(
             raise ValueError("The renderer must be one of the following:{}".format(self.renderer_to_fn.keys()))
         if skip_views and record_file is None:
             raise ValueError("When skip_views is desired, a record_file must be provided.")
-
-        view = self.get_grid_view(views, cell_grid_size=cell_grid_size, grid_size=grid_size)
+        if from_list_of_list:
+            view = self.get_user_defined_grid_view(views)
```

Comment:
Can't we replace it with `isinstance(views[0], list)` instead ? it does not seem name-related to what it does